### PR TITLE
ott 0.33

### PR DESCRIPTION
**Pull request by bacam**

### CODE REVIEW

File: packages/ott/ott.0.33/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+version: "0.33"
```
Comment (palmskog) [inline packages/ott/ott.0.33/opam:2]:
```suggestion
```
Linter complains about this one.

**Comment by kit-ty-kate** [comment]
Do you want to fix ott so that it is compatible with bytecode-only switches or do you want to mark it as incompatible for now?
```
#=== ERROR while compiling ott.0.33 ===========================================#
# context              2.2.0~alpha~dev | linux/arm32 | ocaml-base-compiler.5.0.0 | pinned(https://github.com/ott-lang/ott/archive/0.33.tar.gz)
# path                 ~/.opam/5.0/.opam-switch/build/ott.0.33
# command              ~/.opam/opam-init/hooks/sandbox.sh build make world
# exit-code            2
# env-file             ~/.opam/log/ott-7-df2dbe.env
# output-file          ~/.opam/log/ott-7-df2dbe.out
### output ###
# cd src; make install
# make[1]: Entering directory '/home/opam/.opam/5.0/.opam-switch/build/ott.0.33/src'
# ocamllex grammar_lexer.mll
# 374 states, 16439 transitions, table size 68000 bytes
# 3397 additional bytes used for bindings
# ocamlyacc -v grammar_parser.mly
# 2 rules never reduced
# ocamldep location.ml types.ml auxl.ml merge.ml global_option.ml grammar_lexer.ml grammar_parser.mli grammar_parser.ml version.ml grammar_pp.ml parse_table.ml glr.ml new_term_parser.ml term_parser.ml dependency.ml bounds.ml context_pp.ml quotient_rules.ml grammar_typecheck.ml transform.ml substs_pp.ml subrules_pp.ml embed_pp.ml defns.ml ln_transform.ml coq_induct.ml system_pp.ml lex_menhir_pp.ml align.ml main.ml align.mli bounds.mli coq_induct.mli defns.mli dependency.mli embed_pp.mli grammar_typecheck.mli merge.mli subrules_pp.mli substs_pp.mli system_pp.mli lex_menhir_pp.mli transform.mli term_parser.mli > .depend
# mkdir ../bin
# make opt
# make[2]: Entering directory '/home/opam/.opam/5.0/.opam-switch/build/ott.0.33/src'
# ocamlfind ocamlopt -package str -package ocamlgraph -linkpkg -w y -g -dtypes -unsafe -inline 9 -c location.ml
# ocamlfind: Not supported in your configuration: ocamlopt
# make[2]: *** [Makefile:208: location.cmx] Error 2
# make[2]: Leaving directory '/home/opam/.opam/5.0/.opam-switch/build/ott.0.33/src'
# make[1]: *** [Makefile:229: install] Error 2
# make[1]: Leaving directory '/home/opam/.opam/5.0/.opam-switch/build/ott.0.33/src'
# make: *** [Makefile:37: world] Error 2
```

**Comment by palmskog** [comment]
I'd vote for marking it as incompatible with bytecode-only (but I'll leave the final decision up to @bacam and @tperami).

**Comment by bacam** [comment]
I doubt anyone cares about bytecode-only builds of ott, but the Makefile supports it and appears to work, so it seems a shame not to allow it.

**Comment by bacam** [comment]
That just leaves the old Sail versions, which I'll file a separate PR for.

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**